File: chap11/run-model11-6.R
library(cmdstanr)

d <- read.csv('input/data-switch.csv')
T <- nrow(d)
data <- list(T=T, Y=d$Y)

model <- cmdstan_model('model/model11-6.stan')
fit <- model$sample(data=data, seed=123, parallel_chains=4)
fit$save_object(file='output/result-model11-6.RDS')

File: chap11/run-model11-6.py
import pandas
import cmdstanpy

d = pandas.read_csv('input/data-switch.csv')
T = len(d)
data = {'T':T, 'Y':d.Y}

model = cmdstanpy.CmdStanModel(stan_file='model/model11-6.stan')
fit = model.sample(data=data, seed=123, parallel_chains=4)
fit.save_csvfiles('output/result-model11-6')
